CHANGES.md

```diff
@@ -4,7 +4,34 @@
 
 ### Changed
 
-- rename `POSTGRES_HOST_READER` to `POSTGRES_HOST` in config **breaking change**
+- rename `POSTGRES_HOST_READER` to `PGHOST` in config **breaking change**
+- rename `POSTGRES_USER` to `PGUSER` in config **breaking change**
+- rename `POSTGRES_PASS` to `PGPASSWORD` in config **breaking change**
+- rename `POSTGRES_PORT` to `PGPORT` in config **breaking change**
+- rename `POSTGRES_DBNAME` to `PGDBNAME` in config **breaking change**
```

Comment:
This should change to PGDATABASE, not PGDBNAME

Reply:
good catch